### ISSUE TITLE
Cerberus and The Book Of Hades

### ISSUE THREAD

**Issue by Nexromania** [AlexModGuy/Ice-and-Fire-Suggestions#461]
A Cerberus that spawns in the nether, in a structure near the player's nether portal. It shouldn't be very difficult. After you kill him, you'll find in an altar behind him the book of Hades.
This book when you hold it in your hand or offhand allows you to reanimate dragon corpses and turn thame to life. The skeleton dragon's shoot a withering blast instead of fire or ice. They are weaker than other dragon.

**Comment by MasterOfTheCats** [comment]
The Book of Hades, as described here, seems quite overpowered. So I think it should only be possible to use once, and sacrifice a totem of undying when used. Plus, maybe add special structures in which Ghosts spawn and Cerberus guards? It’ll be loyal to Greek Mythology that way, Cerberus being the guard of the underworld.

**Comment by Nexromania** [comment]
Maybe I didn't explain my idea with much detail, but the skeleton dragons are meant to be very weak like 10-15 hearts max, also be able to only control one at a time. Yeah I guess ghosts that spawn would be cool, maybe not ghosts but something like souls or some kind of spectral entity

**Comment by MasterOfTheCats** [comment]
Ghosts are confirmed to be added in Phase 2, so that works.

**Comment by Nexromania** [comment]
Oh, well in that case yeah, why not?

**Comment by MasterOfTheCats** [comment]
This might not work, but maybe they could even add a naturally-generated portal to the Nether, leading straight into that structure as if it was the entrance to the Underworld.

**Comment by Nexromania** [comment]
Yes, this was part of my idea at first, but fighting the Cerberus just to have access to the Underworld (AKA: Nether) isn't something I see coming in Ice&Fire. That's why I didn't include it here.

**Comment by MasterOfTheCats** [comment]
That’s not what I meant... I thought of a rare naturally generated structure, not replacing normal Minecraft functions like creating Nether Portals (I don’t think that’s possible anyway). You could still get to the Nether, but if you want easier access to the Underworld Palace (randomly thrown name), you could look for the naturally generated structure deep underground in the Overworld, probably at bedrock level. Maybe there’ll be only one Palace in the whole Nether, and Cerberus would respawn every once in a long while, considering he’s an Undead entity.

**Comment by Nexromania** [comment]
Oh. I think this could definitely work. I mean who would want to travel throw the nether, without any directional tools to find the Cerberus. The Portal is a very cool idea, but I'm not too sure that the Cerberus respawning idea is that good at the moment. He not only guards the underworld but the Hades Book too. So why would any player want to kill him again after getting the book from the first time they fight him? Maybe if the book is immovable and you cannot get it with you, than I would see the respawn as useful.

**Comment by Nexromania** [comment]
But the idea needs a lot of work. I only gave the basic idea.

**Comment by MasterOfTheCats** [comment]
I’m sure Alex and Raptor and whoever’s helping them could think of something... 😆

**Comment by KazuyaKan** [comment]
The only problem with this idea is that they are trying to go for more.. general mobs, like cerberus is the name of a specific hellhound, so if "Cerberus" be added then it should be named as Hellhound instead.
